website/docs/r/cs_managed_kubernetes.html.markdown

```diff
@@ -114,6 +114,7 @@ The following arguments are supported:
 * `custom_san` - (Optional, ForceNew, Available in 1.103.2+) Customize the certificate SAN, multiple IP or domain names are separated by English commas (,).
 * `user_ca` - (Optional, ForceNew) The path of customized CA cert, you can use this CA to sign client certs to connect your cluster.
 * `deletion_protection` - (Optional, Available in 1.103.2+)  Whether to enable cluster deletion protection.
+* `enable_rrsa` - (Optional) Whether to enable cluster to support rrsa. Default to `false`. See [RAM Roles for Service Accounts](https://www.alibabacloud.com/help/zh/container-service-for-kubernetes/latest/use-rrsa-to-enforce-access-control).
```

Comment:
`enable_rrsa` - (Optional, Available in xxx)

Reply:
done